File: src/main/kotlin/org/tendiwa/plane/geometry/straightSkeleton/RightSplitNode.java
package org.tendiwa.plane.geometry.straightSkeleton;


import org.tendiwa.geometry.points.Point;

final class RightSplitNode extends SplitNode {
    RightSplitNode(
        Point point,
        OriginalEdgeStart previousEdgeStart,
        OriginalEdgeStart currentEdgeStart
    ) {
        super(point, previousEdgeStart, currentEdgeStart);
    }

    @Override
    boolean isLeft() {
        return false;
    }
}

File: src/main/kotlin/org/tendiwa/plane/geometry/straightSkeleton/RightSplitNode.kt
package org.tendiwa.plane.geometry.straightSkeleton


import org.tendiwa.geometry.points.Point

internal class RightSplitNode(
    point: Point,
    previousEdgeStart: OriginalEdgeStart,
    currentEdgeStart: OriginalEdgeStart) : SplitNode(point, previousEdgeStart, currentEdgeStart) {

    internal override val isLeft: Boolean
        get() = false
}
